src/blocks/main.css

```diff
@@ -0,0 +1,12 @@
+/* #region main */
+
+body {
+  margin: 0;
+}
+
+.main {
```

Comment:
Avoid using tag names for styling, except for `html` and `body`. Consider using a class or ID selector for the `.main` element.